port/unix/omrmmap.c

```diff
@@ -205,11 +210,21 @@ omrmmap_map_file(struct OMRPortLibrary *portLibrary, intptr_t file, uint64_t off
 		spCount++;
 	}
 
-#if defined(J9ZOS39064) && defined(__MAP_64)
+#if defined(J9ZOS39064)
 	if (OMR_ARE_ANY_BITS_SET(flags, OMRPORT_MMAP_FLAG_ZOS_64BIT)) {
-		mmapFlags |= __MAP_64;
+		if (zos_version_at_least(ZOS_V2R4_RELEASE, ZOS_V2R4_VERSION)) {
+			mmapFlags |= __MAP_64;
+		} else {
+			Trc_PRT_mmap_map_file_unix_invalidFlags();
+			errMsg = portLibrary->nls_lookup_message(portLibrary,
+						J9NLS_ERROR | J9NLS_DO_NOT_APPEND_NEWLINE,
```

Comment:
Start the parameter list on a new line: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line.
```suggestion
			errMsg = portLibrary->nls_lookup_message(
						portLibrary,
						J9NLS_ERROR | J9NLS_DO_NOT_APPEND_NEWLINE,
```